js/articles.js

```diff
@@ -0,0 +1,71 @@
+fetch('jsons/articles.json')
+  .then(response => {
+    return response.json()
+  })
+  .then(data => {
+    // Work with JSON data here
+    articlesCounter = data["articles"].length;
+    let a = Math.floor(Math.random() * articlesCounter);
+    let b = Math.floor(Math.random() * articlesCounter);
+    let c = Math.floor(Math.random() * articlesCounter);
+
+    const width = screen.width;
+    const articles = document.getElementsByClassName("articles")[0];
+    if (width>=800){
+      bildArticle(a);
+      bildArticle(b);
+      bildArticle(c);
+    } else if(width<400){
+      bildArticle(a);
+    } else{
+      bildArticle(a);
+      bildArticle(b);
+    }
+
+    function bildArticle(i){
+      let articlesConteiner = document.createElement("div");
+      articlesConteiner.classList.add("articles-conteiner");
+
+      let imgLink = document.createElement("a");
+      imgLink.href = "#article$"+data["articles"][i]["id"];
+
+      let articlesImgConteiner = document.createElement("div");
+      articlesImgConteiner.classList.add("articles-img-conteiner");
+
+      let img = document.createElement("img");
+      img.classList.add("articles-img");
+      img.src = data["articles"][i]["image"];
+      articlesImgConteiner.appendChild(img);
+      imgLink.appendChild(articlesImgConteiner);
+      articlesConteiner.appendChild(imgLink);
+    
+      let articlesDate = document.createElement("span");
+      articlesDate.classList.add("articles-date");
+      articlesDate.innerHTML = data["articles"][i]["date"];
```

Comment:
object literals (it's how ['property'] called) is deprecated.
in that case you could write just
`articlesDate.innerHTML = data["articles"][i].date;`

same things under